mobile/src/main/res/values-de/strings.xml

```diff
@@ -84,6 +84,8 @@
     <string name="action_settings">Einstellungen</string>
     <string name="nfc_dialog_title">Aktion des NFC-Tags</string>
     <string name="info_not_set">Nicht gesetzt</string>
+    <string name="info_settings_moved">Der Menüeintrag Einstellungen wurde in die linke
```

Comment:
Intentional two lines?

Reply:
Nope, thanks for the note, haven't seen that :)